lib/puma/server.rb

```diff
@@ -588,8 +588,8 @@ def normalize_env(env, client)
     end
 
     def default_server_port(env)
-      return PORT_443 if env[HTTPS_KEY] == 'on' || env[HTTPS_KEY] == 'https'
-      env['HTTP_X_FORWARDED_PROTO'] == 'https' ? PORT_443 : PORT_80
+      return PORT_443 if env[HTTPS_KEY] == 'on' || env[HTTPS_KEY] == HTTPS
```

Comment:
this is a bit hard to read as-is. Can we do this instead:

```ruby 
  if ['on', HTTPS].include?(env[HTTPS_KEY]) || env[HTTP_X_FORWARDED_PROTO] == HTTPS 
    PORT_443
  else
    PORT_80
  end
```

Reply:
Thanks for example!